src/extension.ts

```diff
@@ -1298,6 +1299,29 @@ const deleteKubernetes = async (explorerNode?: explorer.ResourceNode) => {
     }
 };
 
+const deleteKubernetesNow = async (explorerNode?: explorer.ResourceNode) => {
```

Comment:
Thanks for the contribution! I noted that this code is almost the same as the function `deleteKubernetes`, with the exception of the kubectl invocation adding `--now`. We should be able to simplify this so as to avoid duplicate code. Take a look at lines 166, and 167 of `extension.ts`, where we create an anonymous function, and call `getEvents`, passing it kubectl, display mode, and the explorer node.

In a similar fashion, you can modify the `deleteKubernetes` function to accept a `DeleteMode` enum (containing `Now`, and `Graceful`), adding the `--now` flag to the kubectl invocation if the if deletion mode is `DeleteMode.Now`.

Reply:
Thank you! Will do!